### ISSUE TITLE
arch "arm" is not built anymore

### ISSUE THREAD

**Issue by zoff99** [TokTok/jvm-toxcore-c#10]
only armv7a is built now.
normal arm .so library is missing.

**Comment by iphydf** [comment]
I don't think we ever built a normal arm .so on Travis. I'm not sure about Jenkins. Do you know what needs to change to build for normal arm?

**Comment by zoff99** [comment]
readelf -A ./armeabi/libtox4j-c.so
Attribute Section: aeabi
File Attributes
  Tag_CPU_name: "ARM v7"
  Tag_CPU_arch: v7
  Tag_CPU_arch_profile: Application
  Tag_ARM_ISA_use: Yes
  Tag_THUMB_ISA_use: Thumb-2
  Tag_FP_arch: VFPv3
  Tag_Advanced_SIMD_arch: NEONv1
  Tag_ABI_PCS_GOT_use: GOT-indirect
  Tag_ABI_PCS_wchar_t: 4
  Tag_ABI_FP_denormal: Needed
  Tag_ABI_FP_exceptions: Needed
  Tag_ABI_FP_number_model: IEEE 754
  Tag_ABI_align_needed: 8-byte
  Tag_ABI_enum_size: int
  Tag_ABI_HardFP_use: SP and DP
  Tag_ABI_optimization_goals: Aggressive Speed
  Tag_CPU_unaligned_access: v6
  Tag_ABI_FP_16bit_format: IEEE 754
  Tag_Virtualization_use: TrustZone

**Comment by zoff99** [comment]
i am trying to find out what the build commands are. i am not really familiar with sbt.
the problem is: 
```
By default, an ARM Clang standalone toolchain will target the armeabi-v7a ABI.
GCC will target armeabi.
Either can be controlled by passing the appropriate -march flag,
and Clang can also be controlled with target.
```

https://developer.android.com/ndk/guides/standalone_toolchain.html

i have googled and did not find how to make clang build for armeabi :-(
why does it need clang? does it work with normal GCC ?

**Comment by iphydf** [comment]
You could try to make it work with GCC. The problem used to be that GCC doesn't know C++, and we need a standard compliant C++ compiler.

**Comment by zoff99** [comment]
oh. thats all. then it should work with gcc just fine.
but lets wait until it even works in Antox (commits are merged) for that switch to GCC

**Comment by iphydf** [comment]
GCC works now. Also, the C++ code is built with cmake now. We might get this to work. How do you build for arm?
